runtime/verbose/errormessagehelper.c

```diff
@@ -973,6 +977,7 @@ printTypeInfoToBuffer(MessageBuffer* buf, U_8 tag, J9UTF8Ref* dataType, BOOLEAN
 		/* Object, init/new object and object 'reference' */
 		printMessage(buf, "'%.*s%.*s%.*s%.*s'",
 				dataType->arity, arrayPrefix,		/* array prefix '[' */
+				// TODO: andrewc Hard assumption
```

Comment:
The stackmap types havent been updated to account for VTs, so this should stay as it is